Jenkinsfile

```diff
@@ -8,11 +8,31 @@ node('us-east-1 && ubuntu && docker && !gpu') {
         checkout scm
         def delim = '"'
         hash = sh(returnStdout: true, script: """curl https://api.github.com/repos/cliqz-oss/cliqz-android/branches/upstream-releases | grep -m 1 sha | cut -d '${delim}' -f 4""").trim()
+        withCredentials([file(credentialsId: 'ceb2d5e9-fc88-418f-aa65-ce0e0d2a7ea1', variable: 'CLIQZ_CI_SSH_KEY')]) {
```

Comment:
would be nice to wrap it in a helper, also to clean it up (with try/finally) after block is executed